addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "<%= htmlComponentName %>",
   "version": "0.0.0",
   "license": "Apache-2.0",
+  "scripts":{
+    "postinstall": "webdriver-manager update"
```

Comment:
I was also under the erroneous impression that `webdriver-manager update` would re-download! Just tested it and it just ain't so. My bad, sorry for wasting your time @juliemr :/

@cironunes this means that we can just run it within `ng test` and it should be perfectly fine.